scripts/src/java/org/oppia/android/scripts/maven/RetrieveLicenseTexts.kt

```diff
@@ -147,16 +150,22 @@ class RetrieveLicenseTexts(
   }
 
   private fun retrieveCopyrightLicense(license: License): CopyrightLicense {
-    val licenseText: String
+    var licenseText: String
     val licenseLink: String
     when (license.verifiedLinkCase) {
       License.VerifiedLinkCase.SCRAPABLE_LINK -> {
         licenseText = fetchLicenseText(license.scrapableLink.url)
         licenseLink = license.scrapableLink.url
+        if (licenseText.length > MAX_CHARS_LIMIT) {
+          licenseText = licenseLink
+        }
       }
       License.VerifiedLinkCase.EXTRACTED_COPY_LINK -> {
         licenseText = fetchLicenseText(license.extractedCopyLink.url)
         licenseLink = license.extractedCopyLink.url
+        if (licenseText.length > MAX_CHARS_LIMIT) {
+          licenseText = licenseLink
+        }
```

Comment:
I don't think this is what we want to do, actually. Can we instead fragment the text into multiple strings & display them in segments in the UI? We intentionally reserve the link for cases when we can't show the text, but in this case we still can with additional work. I think that's important here.

If there is some technical reason we can't do this, though, then the link might be a reasonable back-up.